pkgs/top-level/all-packages.nix

```diff
@@ -9529,7 +9529,9 @@ with pkgs;
   last-resort = callPackage ../data/fonts/last-resort { };
 
   ligo =
-    let ocaml_p = ocaml-ng.ocamlPackages_4_14; in
+    let ocaml_p = ocaml-ng.ocamlPackages_4_14.overrideScope (self: super: {
+      zarith = super.zarith.override { version = "1.13"; };
```

Comment:
This is sad, but Tezos vendors the `.mli` for zarith which makes it very hard to upgrade.